test/tests_basic.jl

```diff
@@ -25,6 +25,10 @@ end
   @test connectivity_pw.num_trees[] isa Integer
   @test_nowarn propertynames(connectivity_pw)
 
+  # `unsafe_wrap`ping a `PointerWrapper`
+  n_vertices::Int = connectivity_pw.num_vertices[]
+  @test unsafe_wrap(Array, connectivity_pw.vertices, (3, n_vertices)) isa Array
```

Comment:
Could you please
- add more than a smoke test, e.g., by checking whether it works to get/set elements
- add more tests with the other allowed arguments